tests/repository/test_determine_repo_dir_clones_repo.py

```diff
@@ -57,12 +58,12 @@ def test_repository_url_with_no_context_file(mocker, template_url):
         repository.determine_repo_dir(
             template_url,
             abbreviations={},
-            clone_to_dir=None,
+            clone_to_dir=user_config_data['cookiecutters_dir'],
```

Comment:
I switched this back now that we're not always checking if there's an existing clone.